recipes/conda_build.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
This is my own script, I've been keeping it in the directory for automatically downloading packages from PyPI and uploading them to my channel on `anaconda.org`.